subsys/shell/backends/shell_adsp_memory_window.c

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2024 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/shell/shell_adsp_memory_window.h>
+#include <zephyr/sys/winstream.h>
+#include <zephyr/logging/log.h>
+
+#include <adsp_memory.h>
+#include <adsp_debug_window.h>
+
+#define _PROMPT "~$ "
```

Comment:
This can be moved to the `Kconfig.backends` file; this will make it consistent with the other backends. Besides that, it looks very good.

For example for the telnet backend:

```
config SHELL_PROMPT_TELNET
	string "Displayed prompt name"
	default "~$ "
	help
	  Displayed prompt name for TELNET backend. If prompt is set, the shell will
	  send two newlines during initialization.
```

Reply:
Oh yes, this should be a Kconfig, now done in v6 push.